client/components/AdminDashboard.js

```diff
@@ -13,10 +17,16 @@ class AdminDashboard extends Component {
         <h3>Admin Dashboard</h3>
         <p> Hello {user.firstName}</p>
         <p>What would you like to do?</p>
-        <ul>
-          <li>Manage Prompts</li>
-          <li>Manage Users</li>
-        </ul>
+        <Card>
+          <CardContent>
+            <Link to={`/admin/all-prompts`}>Manage Prompts</Link>
```

Comment:
it would be good to add the routes to routes.js in the same PR if you are setting them up here

Reply:
nvm